src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+import StyledAvatar from './styled/avatar';
+import StyledImage from './styled/image';
+import StyledText from './styled/text';
+import StyledIcon from './styled/icon';
```

Comment:
Don't change this now for this component. But for future components, we should use an index.js in the styled folder with all the styled-components exported in this only file. Also if there is a complex or huge styled-component, you can put it in another file inside the styled folder and import and export it in the styled index like:
```
export { default as StyledComplexComponent } from './complexComponent';

export const StyledIcon = styled.View<Props>`
    position: absolute;
`;

export const StyledLabel = styled.Text<Props>`
    color: purple;
`;
```
With this approach, we can use only one import statement:
```import { StyledIcon, StyledLabel  } from './styled';```

Reply:
copied!